exercises/matrix/MatrixTest.cs

```diff
@@ -47,10 +47,10 @@ public void Can_extract_column()
     }
 
     [Fact(Skip = "Remove to run test")]
-    public void Can_extract_column_from_non_square_matrix()
+    public void Can_extract_column_from_non_square_matrix_with_no_corresponding_row()
     {
-        var sut = new Matrix("1 2 3\n4 5 6\n7 8 9\n8 7 6");
-        Assert.Equal(new[] { 3, 6, 9, 6 }, sut.Column(3));
+        var sut = new Matrix("1 2 3 4\n 5 6 7 8\n 9 8 7 6");
```

Comment:
I think the problem is the extra space after the 4\n part. Looks like a possible problem with the canonical data.

Reply:
Yup looks like it was a typo:
https://github.com/exercism/problem-specifications/pull/1578/files